spring-cloud-gcp-samples/spring-cloud-gcp-secretmanager-sample/src/main/resources/bootstrap.properties

```diff
@@ -1,15 +1,6 @@
-# You can directly load the secret into a variable like this example
-# This demonstrates multiple ways of loading the same application secret using template syntax.
-#
-# Please refer to the documentation for the protocol syntax.
-#
-# Examples:
-# my-app-secret-3=${sm://projects/<YOUR_PROJECT_ID>/secrets/application-secret}
-# my-app-secret-4=${sm://projects/<YOUR_PROJECT_ID>/secrets/application-secret/version/latest}
-# my-app-secret-5=${sm://<YOUR_PROJECT_ID>/application-secret/latest}
+# Configuration-level settings for Secret Manager must be specified in a bootstrap.properties file because
+# Secret Manager secrets are loaded in the application by Spring Bootstrap Context
 
-# Defaults to latest version if omitted.
-my-app-secret-1=${sm://application-secret}
-
-# Use to specify explicit version (like version 1) under the default project.
-my-app-secret-2=${sm://application-secret/1}
+# spring.cloud.gcp.secretmanager.enabled=<true or false>
```

Comment:
Leave a comment about it being enabled by default when the starter is pulled in?

Reply:
Done.